### DISCUSSION TITLE
Support for raw SQL?

### DISCUSSION THREAD

**Discussion by muqshots** [Brendonovich/prisma-client-rust#57] [Ideas]
Hi!, are there any plans to allow the use of raw statements?

I'd love to use features such as order by random(), but are not yet implemented in the prisma API

**Comment by Brendonovich** [comment]
I did support that before 0.4.1 actually, though I forgot to reimplement it and didn't have any SQL sanitisation. Next update I'll add it back with support for sanitisation.

**Reply by kkharji** [reply]
@Brendonovich is it added as of current release? I'm looking to execute raw sql file content

**Reply by oscartbeaumont** [reply]
https://prisma.brendonovich.dev/extra/raw

**Reply by oscartbeaumont** [reply]
https://prisma.brendonovich.dev/extra/raw

**Reply by kkharji** [reply]
Hey @oscartbeaumont, I saw that, but it didn't work with multiple multi sql statement
execuation.
```
 Execute(Error { is_panic: false, inner: Known(KnownError { message: "Raw query failed. Code:
 `42601`. Message: `db error: ERROR: cannot insert multiple commands into a prepared
 statement`", meta: Object {"code": String("42601"), "message": String("db error: ERROR: cannot
 insert multiple commands into a prepared statement")}, error_code: "P2010" }) })'
```
